### PR TITLE
Feature/df 479 add employment section

### PR DESCRIPTION
**Pull request by 99999989**

### CODE REVIEW

File: alv-portal-ui/src/app/job-publication/job-publication-form/languages/languages.component.ts [hunk 1]
```diff
@@ -3,16 +3,15 @@ import { Observable, of } from 'rxjs';
...
 } from '../../../shared/backend-services/shared.types';
 import { FormArray, FormBuilder, FormGroup, Validators } from '@angular/forms';
 import { FilterLanguageSkill } from '../../../shared/backend-services/candidate/candidate.types';
 
-const EMPTY_LANGUAGE_SKILL: FilterLanguageSkill = {
-  code: null,
-  written: CEFR_Level.NONE,
-  spoken: CEFR_Level.NONE
+const EMPTY_LANGUAGE_SKILL: LanguageSkill = {
+  languageIsoCode: null,
```
Comment (birom) [inline alv-portal-ui/src/app/job-publication/job-publication-form/languages/languages.component.ts]: This breaks the ``isAddLanguageSkillEnabled`` method. Please also change that.

File: alv-portal-ui/src/app/job-publication/job-publication-form/languages/languages.component.ts [hunk 2]
```diff
@@ -78,16 +77,16 @@ export class LanguagesComponent implements OnInit {
...
+      spokenLevel: EMPTY_LANGUAGE_SKILL.spokenLevel
     }, { emitEvent: false });
   }
 
   private createNewLanguageSkillFormGroup(languageSkill = EMPTY_LANGUAGE_SKILL): FormGroup {
     return this.fb.group({
-      code: [languageSkill.code],
-      written: [languageSkill.written],
-      spoken: [languageSkill.spoken]
+      languageIsoCode: [languageSkill.languageIsoCode],
```
Comment (birom) [inline alv-portal-ui/src/app/job-publication/job-publication-form/languages/languages.component.ts]: Please change the ``languages.component.html`` accordingly.

File: alv-portal-ui/src/app/job-publication/job-publication-form/languages/languages.component.ts (hunk 1 above)
Comment (99999989) [reply alv-portal-ui/src/app/job-publication/job-publication-form/languages/languages.component.ts]: Oops... fixed

File: alv-portal-ui/src/app/job-publication/job-publication-form/languages/languages.component.ts (hunk 2 above)
Comment (99999989) [reply alv-portal-ui/src/app/job-publication/job-publication-form/languages/languages.component.ts]: Sorry, I should have run the code at least once after this change. Now it should work.

**Pull request merged**